Shokofin/Sync/UserDataSyncManager.cs

```diff
@@ -233,6 +238,11 @@ public async void OnUserDataSaved(object sender, UserDataSaveEventArgs e)
                     }
                 }
             }
+            catch (HttpRequestException ex) when (ex.StatusCode is HttpStatusCode.Unauthorized) {
+                TryGetUserConfiguration(e.UserId, out var userConfig);
+                if (userConfig is not null)
+                    Logger.LogError(ex, "Invalid or expired API token used. In the plugin settings, please reset the connection to Shoko Server in the user settings section (Jellyfin User={Username})", UserManager.GetUserById(userConfig.UserId).Username);
+            }
```

Comment:
I miss a `return;` statement.

Yes, the `catch` statement at the end of the method body so functionally wise then the `return;` statement would do nothing, but now we have one `catch` without a `return;` and another one with a `return;`. Which, IMO, is inconsistent.

```suggestion
                 return;
            }
```